data/strings/strings.txt

```diff
@@ -23342,54 +23248,54 @@
     zh-Hant = 載入地圖
 
   [pref_tts_speedcams_auto]
-    comment = Speed camera settings menu option - Warn (about speedcams) if risk of exceeding speed limit
-    tags = android
-    en = Warn if there is a risk of exceeding the speed limit
-    af = Waarsku indien daar ’n risiko is dat die grens oorskry word
-    ar = حذرني  إذا هناك احتمالية تجاوز الحد الأقصى للسرعة
-    az = Sürət həddini aşmaq riski varsa, xəbərdarlıq edin
-    be = Папярэджваць калі ёсць рызыка перавышэння дазволенай хуткасці
-    bg = Предупреждава ако има риск от превишаване на ограничението на скоростта
-    ca = Avisa si hi ha risc d'excedir el límit de velocitat
-    cs = Upozorňovat pokud hrozí riziko překročení rychlosti
-    da = Advar hvis der er en risiko for, at hastighedsgrænsen overskrides
-    de = Warnen wenn die Gefahr einer Geschwindigkeitsüberschreitung besteht
-    el = Να ειδοποιούμαι αν υπάρχει κίνδυνος να υπερβώ το όριο ταχύτητας
-    es = Alertar si existe el riesgo de exceder el límite de velocidad
-    es-MX = Avisar si existe el riesgo de exceder el límite de velocidad
+    comment = "Speed cameras" settings menu option (should be short! no more than 47-50 chars) to warn a driver if there is a risk of exceeding the speed limit
+    tags = android,ios
+    en = Warn if exceeding the speed limit
```

Comment:
```suggestion
    en = Warn if speeding
```
(debatable)

Reply:
I've set to `en = Warn if speeding`...

![ios_speed_cam_prefs_en5](https://github.com/organicmaps/organicmaps/assets/25888296/af886f45-7fbc-4239-ba27-140ffdb1e964)